### PR TITLE
add bromine page

### PR DESCRIPTION
**Pull request by Bitbybit123**
I have added the atomic number and symbol for bromine

### CODE REVIEW

**Review by jyyu830 — COMMENTED** [review]
Add description of the element

File: ELEMENTS/Bromine.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: ELEMENT_NAME
+---
+
+Symbol  : Br
+Atomic Number : 35
+Description : It is a volatile red-brown liquid at room temperature that evaporates readily to form a similarly coloured vapour.
```
Comment (kassadym) [inline ELEMENTS/Bromine.md:8]: The rubric requires three sentences about bromine.

File: ELEMENTS/Bromine.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: ELEMENT_NAME
```
Comment (kassadym) [inline ELEMENTS/Bromine.md:3]: ELEMENT_NAME should say Bromine.

File: ELEMENTS/Bromine.md (hunk 1 above)
Comment (Bitbybit123) [reply ELEMENTS/Bromine.md]: Thank you for pointing this out!

**Review by kassadym — APPROVED** [review]
Everything looks good now!

**Comment by Bitbybit123** [comment]
closes #104

**Pull request merged**